ui/core/src/api/utils/SifClient/TendermintSocketPoll.ts

```diff
@@ -0,0 +1,33 @@
+import { EventEmitter2 } from "eventemitter2";
+
+export type TendermintSocketPoll = ReturnType<
+  typeof TendermintSocketPoll
+>;
+
+export function TendermintSocketPoll({ apiUrl }: { apiUrl: string }) {
+  const emitter = new EventEmitter2();
+
+  const fetchNewBlocks = async () => {
+    const blockResponse = await fetch(`${apiUrl}blocks/latest`);
```

Comment:
Other thing I just realised is that we should probably use axios unless we want to install a node-fetch alternative for tests. (Axios is already included as part of the cosmjs lib)